python/dev.py

```diff
@@ -0,0 +1,18 @@
+from get_hatchery_data import get_WDFW_Data, WDFW_to_df
+from create_graph_data import find_distinct_populations, compute_bargraph_data, get_rolling_average
+from data import testData
+
+data = get_WDFW_Data('WALLACE R HATCHERY')
+df = WDFW_to_df(data)
+
+distinct_populations = find_distinct_populations(df)
+
+bargraph = compute_bargraph_data(df, distinct_populations)
+
+rolling_average = get_rolling_average(df, distinct_populations)
+
+with open ("bargraph.json", "w") as outfile:
+    outfile.write(bargraph)
+
+with open ("rolling_average.json", "w") as outfile:
+    outfile.write(rolling_average)
```

Comment:
Nice this is a good idea! Having something to just test an individual hatchery for a provider.

I'm not super inclined to start doing this, but you'd normally have unit tests in a large project!

Example framework https://docs.pytest.org/en/7.3.x/